packages/dcos-history/extra/dcos-history.service

```diff
@@ -8,7 +8,8 @@ Restart=always
 StartLimitInterval=0
 RestartSec=5
 EnvironmentFile=/opt/mesosphere/environment
-EnvironmentFile=-/run/dcos/etc/history-service/history-service.env
```

Comment:
No need for runtime extension. See https://github.com/mesosphere/dcos-enterprise/pull/185/files#diff-8acc8e77a15678f67a353d405fcf971cR549